### PR TITLE
[l10n] Correct quantities in plPL locale

### PR DESCRIPTION
**Pull request by Chriserus**
When it comes to quantities, in polish 'liczba' does not equal 'ilość'.
Same as in English:
"Number refers to how many of something there is: how many mice, how many mutations, or how many calcium channels, for example. In other words, number refers to items we can count.

By contrast, amount refers to how much of something there is: how much stimulation, how much resistance, or how much liquid. In effect, amount refers to quantities we can measure but not individually count."
"Amount" is "ilość" and "number of" is "liczba".
In conclusion, I think it's a welcome small change 👍

### CODE REVIEW

**Comment by DanailH** [comment]
Hi @Chriserus thanks for updating this 😉.

**Review by DanailH — APPROVED** [review]
